grafana-plugin/src/state/rootBaseStore/index.ts

```diff
@@ -178,6 +181,7 @@ export class RootBaseStore {
   async setupPlugin(meta: OnCallAppPluginMeta) {
     this.setupPluginError(null);
     this.onCallApiUrl = getOnCallApiUrl(meta);
+    this.insightsDatasource = meta?.jsonData?.insightsDatasource || 'grafanacloud-usage';
```

Comment:
can `meta` be nullish here? do we need `?`

Reply:
I can't, will update, thanks